.circleci/config.yml

```diff
@@ -99,6 +99,28 @@ jobs:
       - set-up-packages
       - run: npm run env start && npm run test:e2e
 
+  svn-deploy:
+    executor:
+      name: php
+    working_directory: ~/project/svn_deploy
+    steps:
+      - attach_workspace:
+          at: ~/project
+      - checkout
+      - run:
+          command: |
+            source $BASH_ENV
+            sudo apt-get update && sudo apt-get install subversion
+            svn co https://plugins.svn.wordpress.org/genesis-custom-blocks package
+            npm ci && npm run gulp
+            cd package
+            SVN_DIRECTORIES="trunk tags assets"
+            svn stat $SVN_DIRECTORIES | { grep -E '^\?' || true; } | awk '{print $2}' | xargs -r svn add
+            svn stat $SVN_DIRECTORIES | { grep -E '^\!' || true; } | awk '{print $2}' | xargs -r svn rm
```

Comment:
Removes files from version control if we've deleted them. Probably not common, but we might delete or rename JS files.